### PR TITLE
 Use prefix underscore for state variables and no underscore for parameters

### PR DESCRIPTION
**Pull request by elopio**
Requires #1281 and #1291.

- [ ] 📘 I've reviewed the [OpenZeppelin Contributor Guidelines](../blob/master/CONTRIBUTING.md)
- [ ] ✅ I've added tests where applicable to test my new functionality.
- [ ] 📖 I've made sure that my contracts are well-documented.
- [ ] 🎨 I've run the JS/Solidity linters and fixed any issues (`npm run lint:all:fix`).

### CODE REVIEW

File: contracts/access/Whitelist.sol [hunk 1]
```diff
@@ -13,81 +13,81 @@ import "../access/rbac/RBAC.sol";
 contract Whitelist is Ownable, RBAC {
 
   // Name of the whitelisted role.
-  string private constant ROLE_WHITELISTED = "whitelist";
+  string private constant _ROLE_WHITELISTED = "whitelist";
```
Comment (frangio) [inline contracts/access/Whitelist.sol]: I think we should make an exception for constants. :slightly_smiling_face:

Comment (elopio) [reply contracts/access/Whitelist.sol]: Oh, no, why? if there are public constants for some reason, they will not have the underscore. Then you can't tell what's private constant and what is not.

Comment (frangio) [reply contracts/access/Whitelist.sol]: Hm, okay. We can postpone this decision anyway.

**Comment by frangio** [comment]
Waiting for `rbac-migration` branch to be merged.

**Comment by frangio** [comment]
The build passed locally on my computer. Merging manually!

**Pull request merged**